automation/build-bin.ts

```diff
@@ -310,7 +310,7 @@ async function zipPkg() {
 		archive.on('warning', console.warn);
 
 		archive.pipe(outputStream);
-		archive.finalize();
+		archive.finalize().catch(reject);
```

Comment:
It is interesting that the 'no-floating-promises' rule does not insist on promises being awaited, as long as they are handled with `.catch()`.